Resources/Prototypes/Entities/Constructible/Walls/signs.yml

```diff
@@ -0,0 +1,735 @@
+- type: entity
+  id: BaseSign
+  name: base sign
+  abstract: true
+  description: based on what
+  components:
+  - type: Clickable
+  - type: Collidable
+    shapes:
+    - !type:PhysShapeAabb
+      bounds: "-0.1,-0.4,0.1,0.4"
+      layer:
+      - Opaque
+      - Impassable
+      - MobImpassable
+      - VaultImpassable
+      - SmallImpassable
```

Comment:
As you said, you cannot use a parent to add the `Collidable` component: if you redefine the `Collidable` in a child (which you must do because they do not have the same AABB), you'll have to redefine its properties, including all the layers.

The same may apply for the `Sprite` component's `drawdepth`.

So IMO remove the `Collidable` component in the BaseSign. The point of the parent is to remove duplication of the definition of Damageable, Destructible, Clickable ~~and InteractionOutline~~ for all the signs